roles/ceph-validate/tasks/check_iscsi.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: fail if unsupported chap configuration
+  fail:
+    msg: "Mixing clients with CHAP enabled and disabled is not supported."
+  with_items: "{{ client_connections }}"
```

Comment:
`client_connections` is undefined because the var is part of ceph-iscsi, so I guess we either move the vars in ceph-defaults or temporarly do this in this code: `"{{ client_connections | default ({}) }}"`

And we need to add to the `when:`

`- item.status is defined`